src/platforms/posix/drivers/gyrosim/gyrosim.cpp

```diff
@@ -312,7 +312,6 @@ GYROSIM::GYROSIM(const char *path_accel, const char *path_gyro, enum Rotation ro
 	VirtDevObj("GYROSIM", path_accel, ACCEL_BASE_DEVICE_PATH, 1e6 / 400),
 	_gyro(new GYROSIM_gyro(this, path_gyro)),
 	_product(GYROSIMES_REV_C4),
-	_call{},
```

Comment:
Where does this change come from? I don't see `_call` being referenced at all, so should be safe to remove.